src/helpers/PhpDocTypeParser.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Barryvdh\LaravelIdeHelper\helpers;
```

Comment:
Not sure if 'helpers' is appropriate, perhaps something more descriptive like Parsers? But it should follow PSR12 and use StudlyCaps

Reply:
Makes sense, fixed! And I Improved the filename to more accurately reflect it's use.